package.json

```diff
@@ -25,16 +25,17 @@
   },
   "homepage": "https://github.com/crookedneighbor/markdown-it-link-attributes",
   "devDependencies": {
-    "browserify": "^13.0.0",
-    "chai": "^3.5.0",
-    "chalk": "^1.1.3",
+    "browserify": "^14.4.0",
+    "chai": "^4.1.2",
+    "chalk": "^2.1.0",
+    "markdown-it": "^8.4.0",
```

Comment:
I think this needs to be removed from the package.json so the builds can run tests against various version of `markdown-it` 

https://github.com/crookedneighbor/markdown-it-link-attributes/blob/2122a01528724e91acb95d129d1064ada3664f12/.travis.yml#L6

Reply:
emm... sorry for that :(

Reply:
No worries, it's a weird pattern. Couldn't figure out a better way to do it (open to suggestions!)